lib/rock_paper_scissors.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Games.RockPaperScissors do
+  @winning_choices [{"rock", "scissors"}, {"scissors", "paper"}, {"paper", "rock"}]
+
+
+
+      def get_choice do
+        "Scissors"
+        # user_choice = IO.gets("Type Rock, Paper, or Scissors:")
+      end
+
+      def generate_choice do
+        Enum.random(["rock, paper, scissors"])
+      end
+
+      def decide_winner(user_choice, computer_choice)
+          when user_choice == computer_choice do
+        "Tie, suit and tie"
+      end
+
+      def decide_winner(user_choice, computer_choice)
+          when {user_choice, computer_choice} in @winning_choices do
+        "Player wins since #{user_choice} beats #{computer_choice}"
+      end
+
+      def decide_winner(user_choice, computer_choice) do
```

Comment:
The bug is the result of having a default case. That handles when the computer wins.

Data validation with guards could work to ensure there is valid input. You could repeat similar guard logic you use for when the user.